README.md

```diff
@@ -8,6 +9,7 @@
   1chipML
 </h1>
 
+![Build and Tests](https://github.com/felix642/1chipML/actions/workflows/make.yml/badge.svg?style=flat)
```

Comment:
This is cool, but the link is not valid. It should be the main repo, not the fork ;)

Reply:
Good catch